.cirrus/modules/build.star

```diff
@@ -40,11 +40,12 @@ def build_script():
         "source cirrus-env BUILD",
         "source .cirrus/use-gradle-wrapper.sh",
         "regular_gradle_build_deploy_analyze ${BUILD_ARGUMENTS}",
+        "echo 'Checking if any files are uncommitted in the Go code (this may happen to the generated code)'",
+        "git diff --exit-code --name-only -- sonar-helm-for-iac/",
```

Comment:
We don't regenerate Go protobuf definitions automatically. Probably, it can be tuned in `build.gradle.kts` not to forget about it later.

Reply:
Added a dependency between tasks. Shouldn't affect performance much, because these tasks are cacheable